Skip rules_go's nogo analyzer on this CGO external dep.

See patches/mattn_go_sqlite3/no_nogo.patch for rationale.

--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -1,2 +1,3 @@
     importpath = "github.com/godbus/dbus/v5",
+    tags = ["no-nogo"],
     importpath_aliases = ["github.com/godbus/dbus"],
